Days feature

alphonsotran opened the pull request.

alphonsotran committed:
Add conditional render on User page

Add conditional render based on whether current goal exist or not. If exist, then render Day child. If not, then render button that links to new goal form.

alphonsotran committed:
Rearrange scene to normal.

elshaek commented on app/containers/User.js:
```diff
@@ -25,7 +27,7 @@ export default class MainPage extends Component {
     });
 
-    axios.get("https://make-it-happen-api.herokuapp.com/api/day/count")
+    axios.get("https://make-it-happen-api.herokuapp.com/api/days/count")
```
axios.get("https://make-it-happen-api.herokuapp.com/api/days/count", {
        access_token: this.props.accessToken })

elshaek committed:
Update User.js

Add props to constructor and super
Add access_token params to both axios.get days/count and goals/achieved routes
